src/main/bash/copy-images.sh

```diff
@@ -1,27 +1,43 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 DRY_RUN=false; [ "$1" = "--dry-run" ] && DRY_RUN=true && shift
 
 version="$1"
 registry="$2"
-curl -o versions.yaml https://raw.githubusercontent.com/entando-k8s/entando-k8s-controller-coordinator/v${version}/charts/entando-k8s-controller-coordinator/templates/docker-image-info-configmap.yaml
+curl -f -o versions.yaml https://raw.githubusercontent.com/entando-k8s/entando-k8s-controller-coordinator/v${version}/charts/entando-k8s-controller-coordinator/templates/docker-image-info-configmap.yaml
+retcode=$?
+if [ $retcode -ne 0 ]; then
+  echo "Retrieving configmap from Openshift"
+  oc get configmap -o yaml -n entando "entando-docker-image-info-v${1}" > versions.yaml
+fi
 
-! $DRY_RUN && {
+if $DRY_RUN; then
+  "docker pull entando/entando-k8s-controller-coordinator:${version}"
```

Comment:
@ampie are you missing an `echo` here and below?

Reply:
Indeed